Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:xenial
+
+LABEL maintainer="HugeGraph Docker Maintainers <hugegraph@googlegroups.com>"
+
+ENV PKG_URL https://github.com/hugegraph
+
+# 1. Install needed dependencies of GraphServer & RocksDB
+RUN set -x \
+    && apt-get -q update \
+    && apt-get -q install -y --no-install-recommends --no-install-suggests \
+    curl \
+    lsof \
+    g++ \
+    gcc \
+    openjdk-8-jdk \ 
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* 
+
+# 2. Init HugeGraph Sever (speed up)
+RUN set -e \
+    && echo "192.30.253.112 github.com\n151.101.44.249 github.global.ssl.fastly.net" >> /etc/hosts \
+    && mkdir -p /root/server \
+    && curl -L -S ${PKG_URL}/hugegraph/releases/download/v0.10.4/hugegraph-0.10.4.tar.gz -o /root/server.tar.gz \
+    && tar xzf /root/server.tar.gz --strip-components 1 -C /root/server \
+    && rm /root/server.tar.gz \
+    && cd /root/server/ \
+    && sed -i "s/^restserver.url.*$/restserver.url=http:\/\/0.0.0.0:8080/g" ./conf/rest-server.properties \
+    && sed -n '63p' ./bin/start-hugegraph.sh | grep "&" > /dev/null && sed -i 63{s/\&$/#/g} ./bin/start-hugegraph.sh \
+    && sed -n '74p' ./bin/start-hugegraph.sh | grep "exit" > /dev/null && sed -i 74{s/^/#/g} ./bin/start-hugegraph.sh \
+    && ./bin/init-store.sh
```

Comment:
add a shell install-server.sh?

Reply:
Need test it:
if there is a file named `start.sh`
```
echo "old msg"
```
1. `RUN start.sh` 
2. `RUN echo "old msg" && echo "new msg"
If we use a shell file to change the code, would dockerfile think this is a repeated step and skip the rebuild?